UsageRuleSample/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,7 +4,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
-    tools:context=".MainActivity">
+    tools:context="com.rivetz.UsageRuleTUISample.MainActivity">
```

Comment:
Android convention seems to be to ".MainActivity", rather than the full class path.